Framework/script/RepoCleaner/qcrepocleaner/rules/multiple_per_run.py

```diff
@@ -0,0 +1,130 @@
+from collections import defaultdict
+from datetime import datetime
+from datetime import timedelta
+import logging
+from typing import Dict, DefaultDict, List
+
+from qcrepocleaner.Ccdb import Ccdb, ObjectVersion
+from qcrepocleaner.policies_utils import in_grace_period, group_versions
+
+from qcrepocleaner import policies_utils
+
+logger = logging  # default logger
+
+def process(ccdb: Ccdb, object_path: str, delay: int,  from_timestamp: int, to_timestamp: int,
+            extra_params: Dict[str, str]):
+    '''
+    Process this deletion rule on the object. We use the CCDB passed by argument.
+    Objects who have been created recently are spared (delay is expressed in minutes).
+
+    This specific policy, multiple_per_run, operates like this : Keep the first and the last version of a run plus
+    1 version every X minutes.
+
+    Extra parameters:
```

Comment:
`period_pass` and `object_attribute` are not explained as extra parameters

Reply:
yes. and actually `object_attribute` is not used anymore.